src/aind_metadata_service/sharepoint/nsb2023/mapping.py

```diff
@@ -310,22 +308,70 @@ def _parse_concentration_str(
 
     @property
     def aind_age_at_injection(self) -> Optional[Decimal]:
-        """Maps age_at_injection to aind model"""
+        """Maps age_at_injection to aind model."""
         return self._parse_basic_decimal_str(self._nsb.age_at_injection)
 
+    @property
+    def aind_aind_project_name(self) -> Optional[Any]:
+        """Maps aind_project_name to aind model."""
+        return (
+            None
+            if self._nsb.aind_project_name is None
+            else {
+                self._nsb.aind_project_name.BRAIN__WIDE__CIRCUIT__DYN: None,
```

Comment:
should we constrain this list to the actual list of project names?

Reply:
This was automatically generated from the list of options in the request form. . We're not mapping project name right now so this isn't going into the schema.